src/github/githubRepository.ts

```diff
@@ -221,6 +221,13 @@ export class GitHubRepository implements IGitHubRepository, vscode.Disposable {
 			});
 
 			const hasMorePages = !!result.headers.link && result.headers.link.indexOf('rel="next"') > -1;
+			if (!result.data) {

```

Comment:
It feels like a nit pick, but since this is a scenario we can't identify, i think we should log a warning and leave a comment in the code stating why, link to the issue or something.

Reply:
Done, PTAL.